fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -634,4 +641,122 @@ public String getMaterializedViewDdlStmt(boolean simple) {
         return sb.toString();
     }
 
+    public Set<String> getPartitionNamesToRefresh() {
```

Comment:
This name is simiilar with getNeedRefreshPartitionNames, could you rename these two function? eg. getPartitionNamesToRefreshForMV() ,  getChangedPartitionNamesOfTable?

Reply:
ok, done